source/MissionAction.h

```diff
@@ -41,6 +43,18 @@ class UI;
 // In addition to performing a GameAction, a MissionAction can gate the task on
 // the ownership of specific outfits and also display dialogs or conversations.
 class MissionAction {
+private:
+	struct MissionDialog {
```

Comment:
```suggestion
	class MissionDialog {
	public:
```
It's in the style guide